[2024-08-21T22:01:51.467+0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-21T22:01:51.488+0500] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 3_models_dag_home_2.train_hgb scheduled__2024-08-20T01:00:00+00:00 [queued]>
[2024-08-21T22:01:51.497+0500] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 3_models_dag_home_2.train_hgb scheduled__2024-08-20T01:00:00+00:00 [queued]>
[2024-08-21T22:01:51.497+0500] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2024-08-21T22:01:51.513+0500] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): train_hgb> on 2024-08-20 01:00:00+00:00
[2024-08-21T22:01:51.530+0500] {logging_mixin.py:190} WARNING - /home/nicolaspc/anaconda3/envs/mlops_env/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=339171) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-21T22:01:51.531+0500] {standard_task_runner.py:72} INFO - Started process 339333 to run task
[2024-08-21T22:01:51.529+0500] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', '3_models_dag_home_2', 'train_hgb', 'scheduled__2024-08-20T01:00:00+00:00', '--job-id', '221', '--raw', '--subdir', 'DAGS_FOLDER/homework_stepik_2.py', '--cfg-path', '/tmp/tmpdbbonylj']
[2024-08-21T22:01:51.532+0500] {standard_task_runner.py:105} INFO - Job 221: Subtask train_hgb
[2024-08-21T22:01:51.603+0500] {task_command.py:467} INFO - Running <TaskInstance: 3_models_dag_home_2.train_hgb scheduled__2024-08-20T01:00:00+00:00 [running]> on host nicolaspc-HP-Pavilion-Laptop-15-eh1xxx
[2024-08-21T22:01:51.793+0500] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Nicolas Pal' AIRFLOW_CTX_DAG_ID='3_models_dag_home_2' AIRFLOW_CTX_TASK_ID='train_hgb' AIRFLOW_CTX_EXECUTION_DATE='2024-08-20T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-20T01:00:00+00:00'
[2024-08-21T22:01:51.794+0500] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-21T22:01:51.847+0500] {s3.py:1387} INFO - Downloading source S3 file from Bucket test-bucket-nicolas-1 with path dataset/X_train.pkl
[2024-08-21T22:01:51.864+0500] {base.py:84} INFO - Retrieving connection 'aws_default'
[2024-08-21T22:01:51.865+0500] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2024-08-21T22:01:53.702+0500] {s3.py:1387} INFO - Downloading source S3 file from Bucket test-bucket-nicolas-1 with path dataset/X_test.pkl
[2024-08-21T22:01:54.156+0500] {s3.py:1387} INFO - Downloading source S3 file from Bucket test-bucket-nicolas-1 with path dataset/y_train.pkl
[2024-08-21T22:01:54.625+0500] {s3.py:1387} INFO - Downloading source S3 file from Bucket test-bucket-nicolas-1 with path dataset/y_test.pkl
[2024-08-21T22:01:55.046+0500] {homework_stepik_2.py:212} INFO - Данные загружены
[2024-08-21T22:01:57.390+0500] {credentials.py:1147} INFO - Found credentials in environment variables.
[2024-08-21T22:02:09.523+0500] {logging_mixin.py:190} WARNING - 2024/08/21 22:02:09 INFO mlflow.models.evaluation.default_evaluator: Computing model predictions.
[2024-08-21T22:02:09.537+0500] {logging_mixin.py:190} WARNING - /home/nicolaspc/anaconda3/envs/mlops_env/lib/python3.12/site-packages/sklearn/base.py:486 UserWarning: X has feature names, but HistGradientBoostingRegressor was fitted without feature names
[2024-08-21T22:02:09.547+0500] {logging_mixin.py:190} WARNING - /home/nicolaspc/anaconda3/envs/mlops_env/lib/python3.12/site-packages/sklearn/metrics/_regression.py:492 FutureWarning: 'squared' is deprecated in version 1.4 and will be removed in 1.6. To calculate the root mean squared error, use the function'root_mean_squared_error'.
[2024-08-21T22:02:09.550+0500] {logging_mixin.py:190} WARNING - 2024/08/21 22:02:09 INFO mlflow.models.evaluation.default_evaluator: Testing metrics on first row...
[2024-08-21T22:02:09.551+0500] {logging_mixin.py:190} WARNING - 2024/08/21 22:02:09 WARNING mlflow.models.evaluation.default_evaluator: SHAP or matplotlib package is not installed, so model explainability insights will not be logged.
[2024-08-21T22:02:09.575+0500] {python.py:240} INFO - Done. Returned value was: {'start_tiemstamp': '20240821 22:01', 'data_download_start': '20240821 22:01', 'data_download_end': '20240821 22:01', 'data_preparation_start': '20240821 22:01', 'data_preparation_end': '20240821 22:01', 'hgb_train_start': '20240821 22:01', 'hgb_train_end': '20240821 22:01', 'hgb_R^2_score': 0.8378787894932973, 'hgb_rmse': 0.4609176605547525, 'hgb_mae': 0.20147077567488975}
[2024-08-21T22:02:09.606+0500] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-21T22:02:09.606+0500] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=3_models_dag_home_2, task_id=train_hgb, run_id=scheduled__2024-08-20T01:00:00+00:00, execution_date=20240820T010000, start_date=20240821T170151, end_date=20240821T170209
[2024-08-21T22:02:09.670+0500] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-21T22:02:09.692+0500] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-21T22:02:09.696+0500] {local_task_job_runner.py:240} INFO - ::endgroup::
